src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -11,23 +11,31 @@ const chartData = {
       name: "White mice",
       dataPoints: [],
       color: "#f4ce83",
-      maxPoints: 100,
-      fixedMin: 0,
-      expandOnly: true
+      maxPoints: 20,
+      fixedMinA2: 0,
+      fixedMaxA2: 50,
+      expandOnly: true,
```

Comment:
I think `expandOnly` is ignored when `fixedMaxA2` is present, right?

Reply:
yes - not sure if this is correct behavior or whether we should have the "fixed" max as a baseline and, if expand is enabled, expand as required.